bootstrap.py

```diff
@@ -0,0 +1,84 @@
+import random
+import threading
+import uuid
+
+from keystoneclient import exceptions
+from keystoneclient.v3 import client
```

Comment:
should be able to use http://docs.openstack.org/developer/python-keystoneclient/api/keystoneclient.html#keystoneclient.client.Client